apps/bmd/src/AppRoot.tsx

```diff
@@ -690,10 +724,16 @@ const AppRoot: React.FC<Props> = ({
           break
         }
         case 'pollworker': {
+          const isValid =
+            cardData.h === optionalElectionDefinition?.electionHash
+          const possibleCardTally: Optional<CardTally> = isValid
+            ? ((await card.readLongObject()) as Optional<CardTally>)
+            : undefined
```

Comment:
ah so this means that it will wait to declare a poll worker card is present until we read the long value. I can live with that for now, but it will mean that inserting the poll worker card will cause a longer lag before the screen reacts. Let's keep an eye on this when we test on live harware and see how this looks.

One tweak here: no reason to go read the long object if there is no long value stored on the card.

Reply:
This seems to also be how we handle reading the long value on voter cards, in practice I haven't really noticed any major lag.

Reply:
good point. Maybe an optimization we can make later on both.